docs-sources/includes/_Archivist.md

```diff
@@ -35,6 +50,14 @@ a short configuration description for each available vault backends
 alongside a list of supported operations for that backend.
 </aside>
 
+
+### List, read and write order
+
+In `./config/default.yaml`, listOrder, readOrder, and writeOrder properties **have to** be defined to specify the order when reading or writting data.<br>
+For read operations (listOrder, readOrder), mage will attempt the mentioned vaults until one is successful. However, for write operations (writeOrder), mage will write to each mentioned vault, in the given order.
```

Comment:
The system seems to be a bit more elaborate: I *think* `state.archivist.distribute` will run multiple write operations in the configured write order.

https://github.com/mage/mage/blob/master/lib/archivist/index.js#L849

This means two things:

  1. Like you describe, topics configured to write to more than one backend will write to those backends in that order
  2. If you write multiple topics within a given state transaction, then they will be written to their respectively configured vault backend in the ordered provided by configuration